apps/api/src/app/providers/storage/interfaces/storage-provider.interface.ts

```diff
@@ -0,0 +1,20 @@
+export interface StorageProvider {
+  /**
+   * Gets file content in bytes by filename.
+   *
+   * @param key Filename
```

Comment:
Update param description to be more descriptive

"Path to file, relative to root of storage directory"